app.js

```diff
@@ -89,3 +89,18 @@ renderTableData(shopTwo);
 renderTableData(shopThree);
 renderTableData(shopFour);
 renderTableData(shopFive);
+
+var newStoreForm = document.getElementById('new-location');
+newStoreForm.addEventListener('submit', function(event) {
+  event.preventDefault();
+  if(!event.target.store.value && !event.target.newMin.value && !event.target.newMax.value && !event.target.newAvg.value) {
```

Comment:
@azizaalhouni @kaylynyuh great use of event.target, along with the use of the && operator!